### ISSUE TITLE
v1.3.0 seems broken for modify / drag

### ISSUE THREAD

**Issue by bartvde** [mapbox/mapbox-gl-draw#1055]
Because of the upgrade to geojson-extent [1] I think we are running into: https://github.com/mapbox/geojson-extent/issues/14

[1] https://github.com/mapbox/mapbox-gl-draw/commit/c12fb64b90d00e877e94b12b60bc3b80e42924dd

![Screenshot from 2021-05-27 12-58-53](https://user-images.githubusercontent.com/319678/119814978-548c3980-beeb-11eb-876e-ba619686d40a.png)

cc @mourner

**Comment by ThisIsMissEm** [comment]
I've been looking into this too, and it looks like v1.3.0 actually also introduced other breaking changes; The error there is misleading but I'm not sure why. That package does still exist, it's just not loaded correctly.

It seems to be related to the upgrade to `@mapbox/geojson-extent`, I've currently mapbox-gl-draw@1.3.0 working with mapbox-gl@1.13.0 with the following added to my package.json, which forces that dependency to be downgraded:
```"resolutions": {
    "@mapbox/geojson-extent": "^0.3.2"
  },```

**Comment by bartvde** [comment]
yeah in v1 they changed the exports in that package (geojson-extent)

**Comment by ThisIsMissEm** [comment]
Related: 
- https://github.com/mapbox/geojson-extent/issues/14
- https://github.com/mapbox/geojson-coords/pull/2

**Comment by SnailBones** [comment]
@bartvde Can you share instructions on how to duplicate this bug?

**Comment by bartvde** [comment]
@SnailBones sure I will provide them today

**Comment by bartvde** [comment]
okay finally got it reproduced, I think it's related to custom draw modes actually, I couldn't reproduce it in vanilla mapbox-gl-draw modes.

See: https://github.com/bartvde/mapbox-react-examples/pull/1

And: 
![Uploading Peek 2021-09-13 17-35.gif…]()

**Comment by SnailBones** [comment]
Thanks @bartvde! It looks like [this fix](https://github.com/mapbox/geojson-coords/pull/9) was merged into geojson-coords, but it hasn't yet been released on NPM. Working on that now!
